oxygen.md

```diff
@@ -510,7 +510,7 @@ This test looks for EGO oxygen data in the same phase (ascending or descending f
 
 ### Bad P/T/S QC spreading
 
-The test checks that the dissolved oxygen concentration in /mumol/kg is computed from a valid pressure, temperature and salinity. 
+The test checks that the dissolved oxygen concentration in µmol/kg is computed from a valid pressure, temperature and salinity. 
```

Comment:
At the moment we have issues with some symbols in all 3 formats markdown, pdf and html thus I changes it to /mumol/kg for now which works to build the pdf.

Reply:
I will do this quickly and close this